### PR TITLE
Handle multiple packages with same PackageId

### PR DESCRIPTION
**Pull request by ajaybhargavb**
#8 
@Eilon @pranavkm

### CODE REVIEW

File: src/NuGetPackageVerifier/IssueProcessor.cs [hunk 1]
```diff
@@ -10,28 +10,30 @@ namespace NuGetPackageVerifier
 {
     public class IssueProcessor
     {
+        List<IssueIgnore> allIssuesToIgnore;
```
Comment (Eilon) [inline src/NuGetPackageVerifier/IssueProcessor.cs]: `private` and `_`?

Comment (ajaybhargavb) [inline src/NuGetPackageVerifier/IssueProcessor.cs]: Yes, will update.

File: src/NuGetPackageVerifier/Program.cs [hunk 2]
```diff
@@ -140,38 +140,34 @@ public static int Main(string[] args)
                         totalErrors++;
                         continue;
                     }
-                    if (packagesWithId.Count() > 1)
-                    {
-                        logger.LogError("Found more than one package with id '{0}' in the repo", packageId);
```
Comment (Eilon) [inline src/NuGetPackageVerifier/Program.cs]: Yay for removing this :smile:

**Comment by Eilon** [comment]
:shipit:

File: src/NuGetPackageVerifier/IssueProcessor.cs [hunk 3]
```diff
@@ -10,28 +10,30 @@ namespace NuGetPackageVerifier
...
+
         public IssueProcessor(IEnumerable<IssueIgnore> issuesToIgnore)
         {
-            IssuesToIgnore = issuesToIgnore?.ToList();
+            allIssuesToIgnore = issuesToIgnore?.ToList();
+            RemainingIssuesToIgnore = new List<IssueIgnore>(allIssuesToIgnore);
         }
 
-        public List<IssueIgnore> IssuesToIgnore { get; private set; }
+        public List<IssueIgnore> RemainingIssuesToIgnore { get; private set; }
```
Comment (pranavkm) [inline src/NuGetPackageVerifier/IssueProcessor.cs]: `public List<IssueIgnore> RemainingIssuesToIgnore { get; }`

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**